listeners/types.go

```diff
@@ -0,0 +1,23 @@
+package listeners
+
+import (
+	"github.com/planetdecred/dcrlibwallet"
+)
+
+type TxNotifType int
+
+const (
+	// Transaction notification types
+	NewTransaction TxNotifType = iota // 0 = New transaction.
+	BlockAttached                     // 1 = block attached.
+	TxConfirmed                       // 2 = Transaction confirmed.
+)
+
+// TxNotification models transaction notifications.
+type TxNotification struct {
+	NotificationType TxNotifType
```

Comment:
```suggestion
type TxNotification struct {
	Type TxNotifType
```

To read as `TxNotification.Type` instead of `TxNotification.NotificationType`.